spec/n1ql_spec.rb

```diff
@@ -62,6 +69,33 @@ class N1QLTest < CouchbaseOrm::Base
         }
 
         expect(Set.new(docs)).to eq(Set.new(%w[bob jane]))
+
```

Comment:
maybe should be split in another it bloc, target good practice is one expect per it